src/index.css

```diff
@@ -22,12 +23,15 @@
   --scrollbar-border-color: #999;
   --scrollbar-border-radius: 0;
   --scrollbar-box-shadow: none;
+  --scrollbar-hover-z-index: 30;
```

Comment:
Can you update it to `--scrollbar-z-index-hover` to make it consistent with other `hover` CSS properties ?